src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="/images/logo.png"
+          alt="logo-company"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple"
+            class="nav-link is-active">APPLE</a>
+          </li>
+          <li class="nav-item">
+            <a href="#samsung"
+            class="nav-link">SAMSUNG</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav-item">
            <a href="#apple" class="nav-link is-active">APPLE</a>
          </li>

          <li class="nav-item">
            <ahref="#samsung"class="nav-link">SAMSUNG</a>
          </li>
```

Add blank lines between multiple sibling blocks

FIx everywhere